modules/social_features/social_event/src/EdaHandler.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Drupal\social_event;
+
+use CloudEvents\V1\CloudEvent;
+use Drupal\Component\Uuid\UuidInterface;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface;
+use Drupal\social_eda\DataFormatter as SocialEdaDataFormatter;
+use Drupal\social_eda_dispatcher\Dispatcher as SocialEdaDispatcher;
+use Drupal\social_event\Entity\Node\Event;
+use Drupal\social_event\Event\EventCreateEventData;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Eda Handler class.
```

Comment:
Lets give this a slightly clearer description than repeating the class declaration.

```suggestion
 * Handle's hook invocations for EDA related operations of the event entity.
```